libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -79,7 +79,7 @@ const AP_Param::GroupInfo AP_AHRS::var_info[] = {
 
     // @Param: WIND_MAX
     // @DisplayName: Maximum wind
-    // @Description: This sets the maximum allowable difference between ground speed and airspeed. This allows the plane to cope with a failing airspeed sensor. A value of zero means to use the airspeed as is. See ARSPD_OPTIONS and ARSPD_MAX_WIND to disable airspeed sensors.
+    // @Description: This sets the maximum allowable difference between ground speed and airspeed. A value of zero means to use the airspeed as is. This allows the plane to cope with a failing airspeed sensor by ignoring it. See ARSPD_OPTIONS and ARSPD_WIND_MAX to disable airspeed sensors.
```

Comment:
does not ignore it...clips it to limit max wind difference

Reply:
Ah well good I wrote that then - how about "This allows the plane to cope with a failing airspeed sensor by clips it to groundspeed plus the  max wind limit."